### PR TITLE
[AZURE] documentation on updated azure in gitops

### PR DESCRIPTION
**Pull request by shivani170**
# Description
Updated documentation
Fixes: https://github.com/devtron-labs/devtron/issues/462

### CODE REVIEW

File: docs/user-guide/global-configurations/gitops.md [hunk 1]
```diff
@@ -1,47 +1,51 @@
...
-2. Github Organization Name / Gitlab Group id 
-3. Git access credential
+1. Git Host / Azure DevOps Organisation Url <br />
+2. Github Organization Name / Gitlab Group id / Azure DevOps Project Name <br />
+3. Git access credential <br />
 
 ### 1. Git Host: 
 
-This field is filled by default, Showing the URL of the selected git providers. For example- https://github.com for Github, https://gitlab.com for GitLab.
+This field is filled by default, Showing the URL of the selected git providers. For example- https://github.com for Github, https://gitlab.com for GitLab, https://dev.azure.com/ for Azure.
```
Comment (pghildiyal) [inline docs/user-guide/global-configurations/gitops.md:31]: add
Please change them if they are not the url you use.

Comment (shivani170) [reply docs/user-guide/global-configurations/gitops.md:31]: Screenshot 2021-05-24 at 3 24 22 PM
we are using this link 
@pghildiy

Comment (shivani170) [reply docs/user-guide/global-configurations/gitops.md:31]: yes we are using this link `host: "https://dev.azure.com/"`
@pghildiy

Comment (shivani170) [reply docs/user-guide/global-configurations/gitops.md]: added

**Review by pghildiyal — APPROVED** [review]

**Review by nishant-d — APPROVED** [review]

**Pull request merged**